src/relay/analysis/type_solver.cc

```diff
@@ -57,13 +57,21 @@ class TypeSolver::Reporter : public TypeReporterNode {
     return true;
   }
 
-  TVM_DLL void SetLocation(const ObjectRef& ref) final { location = ref; }
+  TVM_DLL void SetLocation(const Span& span) final { this->span = span; }
```

Comment:
```suggestion
  TVM_DLL void SetSpan(const Span& span) final { this->span = span; }
```
I'm ok with fixing this later but they should be symmetric

Reply:
Yeah looks like something I missed